From 4ed3c44131d3047b7638c259f8fad1b9a6db8546 Mon Sep 17 00:00:00 2001
From: Gwenael Treuveur <gwenael.treuveur@foss.st.com>
Date: Tue, 27 Aug 2024 17:22:40 +0200
Subject: [PATCH] arm64: defconfig: addition of STM32_M0_RPROC in fragment-02

Addition of the STM32 M0 RPROC driver within
fragment-02-defconfig-addons.config.

Change-Id: Ic60efb78302a355092fb4ecb0d7c06e855b2584d
Signed-off-by: Gwenael Treuveur <gwenael.treuveur@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/400538
Tested-by: Gwenael TREUVEUR <gwenael.treuveur@st.com>
Domain-Review: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Reviewed-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Gwenael TREUVEUR <gwenael.treuveur@st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -97,6 +97,7 @@ CONFIG_STM32_HYPERBUS=y
 CONFIG_STM32_I3C_MASTER=m
 CONFIG_STM32_IPCC=y
 CONFIG_STM32_LPTIMER_CNT=m
+CONFIG_STM32_M0_RPROC=y
 CONFIG_STM32_MDF_ADC=m
 CONFIG_STM32_MDF_CORE=m
 CONFIG_STM32_OMI=y
